src/runfileExample.ini

```diff
@@ -73,6 +73,18 @@ density = rho
 uvw = 'u v w'
 pressure = p
 
+[heatSource]
+numHeatSources = 1
+
+[heatSource1]
+# value of heat source
+# if all units follow International System, [value] = watts/m^3
```

Comment:
what does the "if" imply?

Reply:
You're right. We have SI everywhere

Reply:
ok, i'll remove the "if" from the comment and then land this; thanks for knocking it out.